lib/good_bread/accounts.ex

```diff
@@ -30,4 +34,27 @@ defmodule GoodBread.Accounts do
   def change_user(%User{} = user, attrs \\ %{}) do
     User.changeset(user, attrs)
   end
+
+  # SUBSCRIPTIONS
+
+  def create_subscription(attrs \\ %{}) do
+    %Subscription{}
+    |> Subscription.changeset(attrs)
+    |> Repo.insert()
+  end
+
+  def list_subscriptions do
+    Repo.all(Subscription)
+    |> Repo.preload(:user)
```

Comment:
I'm preloading the user here and preloading the subscription because there is only one so it shouldn't slow down the query too much